github/github.go

```diff
@@ -681,6 +683,36 @@ func (r *ErrorResponse) Error() string {
 		r.Response.StatusCode, r.Message, r.Errors)
 }
 
+func (r *ErrorResponse) Is(target error) bool {
+	v, ok := target.(*ErrorResponse)
+	if !ok {
+		return false
```

Comment:
I'm not sure if you can see all the warnings I see on GitHub from Codecov, but please add a unit test that covers this line.  I'll highlight the others in case you can't see them.

Reply:
I have addressed all code coverage issues with additional tests.